src/vivarium_census_prl_synth_pop/components/observers.py

```diff
@@ -1016,6 +1057,7 @@ def get_observation(self, event: Event) -> pd.DataFrame:
             ],
             additional_key="joint_filing_1040",
         )
+        # todo: add function here to copy each of the necessary columns
```

Comment:
I don't understand this todo. isn't that what copy_from_household_member does?

Reply:
I forgot to remove it.